RNTester/js/examples/PointerEvents/PointerEventsExample.js

```diff
@@ -191,6 +243,18 @@ const exampleClasses: Array<ExampleClass> = [
     description:
       "`box-only` causes touch events on the container's child components to pass through and will only detect touch events on the container itself.",
   },
+  {
+    Component: OverflowVisibleExample,
+    title: '`overflow: visible`',
+    description:
+      '`overflow: visible` style should allow subelements that are outside of the parent box to be touchable.',
```

Comment:
To make the test a bit more obvious, you might want to just add:

> Tapping the parts of Box B outside Box A should print "B touched" and "A touched", and tapping Box C should also print "C touched" and "A touched"

Reply:
Done!